packages/@react-stately/checkbox/docs/useToggleState.mdx

```diff
@@ -0,0 +1,40 @@
+<!-- Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. -->
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import docs from 'docs:@react-stately/toggle';
+import {ClassAPI, HeaderInfo, TypeContext, FunctionAPI, TypeLink} from '@react-spectrum/docs';
+import packageData from '@react-stately/toggle/package.json';
+
+---
+category: Forms
+keywords: [toggle, checkbox, switch, input, state]
+---
+
+# useToggleState
```

Comment:
Was this supposed to be docs for `useCheckboxGroupState`? I believe docs for `useToggleState` is already in `@react-stately/toggle`.

Reply:
Yeah, when creating `@react-stately/checkbox` directory I've just copied the other one as a template and haven't noticed that docs directory got added. I've tweaked the content so it points to the correct package right now.